dex/ltc/params.go

```diff
@@ -0,0 +1,47 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org
+
+package ltc
```

Comment:
We have `dex/{btc,dcr,ltc}`, so we might as well go for a subdirectory. `networks` maybe?